.circleci/config.yml

```diff
@@ -1,6 +1,24 @@
 version: 2.1
 
 jobs:
+  check_pr:
+    docker:
+      - image: cimg/python:3.7.4
+    steps:
+      - checkout
+      - run:
+          name: PR
+          command: |
+            echo "This is a PR"
+  check_merge:
+    docker:
+      - image: cimg/python:3.7.4
+    steps:
+      - checkout
+      - run:
+          name: Merge
+          command: |
+            echo "This is a Merge"
```

Comment:
Shall we delete these two jobs? It looks like they are wasting ~20s on initializing the docker just for printing out a message. Their associated condition filters in workflows can be moved to the next jobs.

Reply:
Deleted. Didn't realize it can take such a long time.